elm/README.md

```diff
@@ -20,3 +20,37 @@ In the private area, you'll see the contents for the logged in user and you can
 
 - add new contents with their tags;
 - filter the shown contents by tag.
+
+## Specification
+
+The `spec` folder contains some end-to-end acceptance tests written using [Quickstrom](https://quickstrom.io/).
+
+To run them, you could execute the following commands inside the `spec` folder, given your application is exposed on `localhost:8000`:
```

Comment:
I am guessing you are actually talking of the `elm/spec` directory.

Reply:
yes, better specify to avoid confusion